config/environments/development.rb

```diff
@@ -15,7 +15,7 @@
   config.consider_all_requests_local = true
 
   # Enable/disable caching. By default caching is disabled.
-  if Rails.root.join('tmp/caching-dev.txt').exist?
+  if Rails.root.join('path', 'to').exist?
```

Comment:
it's correct now. Anyway you could have tried both pieces in `rails c` and checking that they are in fact equivalent